### DISCUSSION TITLE
2021/containerlab-your-network-centric-labs-with-a-docker-ux/

### DISCUSSION THREAD

**Discussion by giscus** [hellt/netdevops.me#19] [Show and tell]
# 2021/containerlab-your-network-centric-labs-with-a-docker-ux/

Applying DevOps to networks.

https://netdevops.me/2021/containerlab-your-network-centric-labs-with-a-docker-ux/

**Comment by f2ka07** [comment]
Containerlab is great addition to  <a href="https://solid-future.com/networking/cisco-certified-devnet-associate-netdevops/">NetDevOps </a> toolset. It is what docker-compose would be if it was created with networking topologies in mind.
